dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/AbstractJdbcEventAnalyticsManager.java

```diff
@@ -991,6 +992,12 @@ else if ( isDouble && !Double.isNaN( (Double) value ) )
             {
                 addGridDoubleTypeValue( (Double) value, grid, header, params );
             }
+            else if ( value instanceof BigDecimal )
+            {
+                // Special handling of double is here not of use. Option in OptionSet can be only floating type 'Number' which is internally Double.
```

Comment:
I think you can remove this part of the comment: "// Special handling of double is here not of use. "

Reply:
done